config/analytics.yml

```diff
@@ -190,6 +190,8 @@ shared:
     - created_at
     - updated_at
     - account_closed_on
+    - account_merge_confirmation_code
+    - account_merge_confirmation_code_generated_at
```

Comment:
If we need this info in analytics, I think we should only send the timestamp (so we can trace how many accounts have a pending merge requested?), not the code itself (that acts as a kind of a password).